src/nvim/ex_getln.c

```diff
@@ -2355,9 +2355,9 @@ static int command_line_changed(CommandLineState *s)
     garray_T save_view;
     win_size_save(&save_view);     // Save current window sizes
     save_search_patterns();        // Save search patterns
-    State |= PREVIEW_EXEC;         // Set live preview execution state
+    State |= CMDPREVIEW_EXEC;         // Set live preview execution state
     do_cmdline(ccline.cmdbuff, NULL, NULL, DOCMD_KEEPLINE|DOCMD_NOWAIT|DOCMD_PREVIEW);
-    State &= ~PREVIEW_EXEC;        // Unset live preview execution state
+    State &= ~CMDPREVIEW_EXEC;        // Unset live preview execution state
```

Comment:
nit: let's keep the comments aligned if we can

Reply:
Done